nautobot/core/api/metadata.py

```diff
@@ -210,6 +221,12 @@ def get_list_display_fields(self, serializer):
         serializer_meta = getattr(serializer, "Meta", None)
         return list(getattr(serializer_meta, "list_display_fields", []))
 
+    def get_advanced_tab_fields(self, serializer):
+        """Try to get the advanced tab fields or default to an empty list."""
+        default_advanced_fields = ["id", "url", "composite_key", "created", "last_updated"]
+        advanced_fields = [field for field in default_advanced_fields if field in serializer.fields.keys()]
+        return advanced_fields
+
```

Comment:
Why are the advanced tab fields hard-coded here but the detail view tabs are defined in the serializer meta?